Mathlib/RingTheory/HahnSeries/PowerSeries.lean

```diff
@@ -169,6 +175,11 @@ def toMvPowerSeries {σ : Type*} [Finite σ] : HahnSeries (σ →₀ ℕ) R ≃+
 
 variable {σ : Type*} [Finite σ]
 
+-- TODO : generalize to all (?) rings of Hahn Series
+/-- If R has no zero divisors, then `HahnSeries (σ →₀ ℕ) R` has no zero divisors, for finite `σ` -/
```

Comment:
Strange. [bors CI seems to think this line is over the 100 character limit](https://github.com/leanprover-community/mathlib4/actions/runs/10905668118/job/30265082041#step:11:2484), but it seemed to be fine here.